settings.gradle

```diff
@@ -57,6 +57,7 @@ include 'clients',
     'connect:runtime',
     'connect:test-plugins',
     'connect:transforms',
+    'coordinator-common',
```

Comment:
I guess the next coordinator-related module is called `share-coordinator`, and there is existent module called `group-coordinator`. Hence, have we consider the name: `common-coordinator`? Or you want to align the naming with `server-common`

Reply:
Mainly 2 reasons for the current name.

1. The new parent coordinator module does not do anything useful on its own. `common-coordinator` has the connotation that this module serves a well defined purpose like group coordinator but that is not the case. It just runs the framework classes and the useful work is only done when a cooperating coordinator provides concrete implementations. You are also right in pointing out the server-common nomenclature. But in the same spirit common-server has a very different connotation and seems incorrect.
2. The original author of the group-coordinator suggested the name.

Reply:
@smjn thanks for sharing. sounds good